edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -30,18 +30,11 @@ cuid=$(id -u)
 
 if [ $cuid -eq 0 ]
 then
-
   # Create the agent user id if it does not exist
   if ! getent passwd "${TARGET_UID}" >/dev/null
   then
-    echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Creating UID ${TARGET_UID} as agent${TARGET_UID}"
-    # Use "useradd" if it is available.
-    if command -v useradd >/dev/null
-    then
```

Comment:
is this check not needed?

Reply:
Yeah, it's not necessary. The docker containers only contain adduser, so we'll never use useradd. And we exclusively use adduser elsewhere, so I don't think we need to use useradd here.